scenario-api/src/main/scala/pl/touk/nussknacker/engine/spel/Implicits.scala

```diff
@@ -1,9 +1,53 @@
 package pl.touk.nussknacker.engine.spel
 
 import pl.touk.nussknacker.engine.graph.expression.Expression
+import pl.touk.nussknacker.engine.spel.SpELImplicits.{EmptyList, EmptyRecord, Input}
 
+import java.util
 import scala.language.implicitConversions
 
+object SpELImplicits {
+  val Input: String = "#input"
+  val EmptyRecord: String = "{:}"
+  val EmptyList: String = "{}"
+}
+
+trait SpELImplicits {
+
+  import collection.JavaConverters._
+
+  def toSpEL(data: Any): String = {
+    def convertCollection(data: List[String]) = s"""{${data.mkString(",")}}"""
+
+    data match {
+      case map: collection.Map[String@unchecked, _] if map.isEmpty =>
+        EmptyRecord
+      case map: collection.Map[String@unchecked, _] =>
+        val elements = map.map{case (key, value) => s""""$key": ${toSpEL(value)}"""}
+        convertCollection(elements.toList)
+      case map: util.Map[String@unchecked, _] =>
+        toSpEL(map.asScala)
+      case collection: Traversable[_] if collection.isEmpty =>
+        EmptyList
```

Comment:
It's not the point of that :) Take a look - you have two cases for `collection: Traversable`. For maps it was needed because empty map is represented as a special literal `{:}` in spel. But for lists - it is just like for one element but without this element ;) Remove this case and you'll see that everything will work as expected.

Reply:
Ahh, ok! Chagned.